tests/Unit/Scraper/Application/ConfiguratorTest.php

```diff
@@ -319,10 +319,12 @@ public function whenUseSomeOldXpathButNotFoundNewsItShouldLogItAndResetVariant()
         $crawler->shouldReceive('getNode')
             ->with(0)
             ->andReturn($rootElement);
-        $crawler->shouldReceive('filterXpath->count')
```

Comment:
Is not possible to use the old way?

Reply:
No, there is an issue with mockery because "filterXpath" functions returns as static.